### PR TITLE
Add the T33249_idic_KCSLanPrint test case.

### PR DESCRIPTION
**Pull request by helloarys**
@fub2 @PaynePei @turtle-fly @JuneZhou19

### CODE REVIEW

File: case/infrasim/virtual_node/T33249_idic_KCSLanPrint.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            if rsp['exitcode'] == 0:
+                self.log('INFO',
+                         'Able to issue local IPMI command ipmitool lan')
+            else:
+                self.result(FAIL,
+                            'Failed to issue local IPMI command ipmitool lan, return: {}'.format(
+                                rsp))
+
+    def deconfig(self):
+        self.log('INFO', 'Deconfig')
```
Comment (turtle-fly) [inline case/infrasim/virtual_node/T33249_idic_KCSLanPrint.py]: @helloarys you need to umount this disk image in deconfig. Or sub-sequence test may random another node while disk image is locked by this test_node.

Comment (helloarys) [inline case/infrasim/virtual_node/T33249_idic_KCSLanPrint.py]: Ok, I will fix this bug.

**Comment by helloarys** [comment]
@turtle-fly @fub2 Already added the unmount process in deconfig. Also add the T33251 test case in this pull request.

File: case/infrasim/virtual_node/T33251_idic_KCSSelList.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "dhcp_server": "192.168.128.4",
+        "dhcp_username": "root",
+        "dhcp_password": "cisolution",
+        "disk_image": "ubuntu_vbmc.vmdk",
+        "host_username": "june",
+        "host_password": "111111"
+    }
+]
```
Comment (turtle-fly) [inline case/infrasim/virtual_node/T33251_idic_KCSSelList.json]: Last new line in code. For git's favor.

**Comment by turtle-fly** [comment]
@helloarys generally OK, but please add a last line new line in the json file.

**Comment by helloarys** [comment]
@turtle-fly Done~

**Comment by turtle-fly** [comment]
:+1:

**Pull request merged**